### PR TITLE
add some tests

### PR DESCRIPTION
**Pull request by jonashuckestein**

### CODE REVIEW

**Comment by jonashuckestein** [comment]
ci rebuild

**Comment by jonashuckestein** [comment]
ci rebuild

**Comment by jonashuckestein** [comment]
ci rebuild

**Comment by jonashuckestein** [comment]
ci rebuild

File: example/main.go [hunk 1]
```diff
@@ -1,53 +1,28 @@
...
-	"github.com/b2aio/typhon/client"
 	"github.com/b2aio/typhon/server"
-	log "github.com/cihub/seelog"
-	"github.com/golang/protobuf/proto"
 
 	"github.com/b2aio/typhon/example/handler"
-	"github.com/b2aio/typhon/example/proto/hello"
 )
 
+func initExampleServer() {
```
Comment (mattheath) [inline example/main.go]: what does this do?

File: server/endpoint_registry_test.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package server
+
+import "testing"
+
+func TestGet(t *testing.T) {
+
+}
```
Comment (mattheath) [inline server/endpoint_registry_test.go]: Can we delete this if its not doing anything?

File: typhon_test.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	})
+
+	// Register an example endpoint
+	server.RegisterEndpoint(handler.Hello)
+	server.RegisterEndpoint(handler.CallHello)
+
+	go server.Run()
+
+	select {
+	case <-server.DefaultServer.NotifyConnected():
```
Comment (mattheath) [inline typhon_test.go]: We should probably add a `NotifyConnected()` func at the package level that operates on the `DefaultServer` to streamline this - we can do separately tho.

**Comment by mattheath** [comment]
Couple of minor things which I _think_ can be deleted, then good to go! :ship:

**Comment by jonashuckestein** [comment]
ci rebuild

**Comment by mattheath** [comment]
:boom:

**Pull request merged**